### ISSUE TITLE
Add additional instructions for external workshops

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/training-specific-template#147]
As part of #143, I left a placeholder for adding back in any additional instructions for external users: https://github.com/AlexsLemonade/training-specific-template/tree/template-refresh#additional-instructions-for-externally-hosted-workshops

At this point, I'm not entirely sure what else we might want to tell those folks! The current README seems pretty general.. The only part I can think of now is how they might want to coordinate a `training-modules` release with the Data Lab. 

So, let's leave this issue here to collect thoughts as we have them. It would be nice if we think of these before closing out the refresh epic, but I don't think it's necessarily _critical for the **epic itself**_.

**Comment by sjspielman** [comment]
I created a repo from this template in my `tardigrades4life` account. I set the visibility to public, which the instructions already indicate is necessary since I am in a free account.

Then, I went run the workflow to create all the setup issues. It failed, b/c I did not have these settings turned on. Once turned on, [it worked](https://github.com/tardigrades4life/template-test/issues)! We should therefore add this to setup - these options must be set in `Settings` -> `Actions`, scroll down to bottom:

<img width="750" alt="Screenshot 2023-09-26 at 2 54 37 PM" src="https://github.com/AlexsLemonade/training-specific-template/assets/4701111/73c1396e-08e6-4c7a-b911-08f9c6187031">

The workflow to copy completed notebooks wasn't actually showing up, which is almost certainly because it's not yet in the `main` branch, so we don't expect it to show up! We should circling back to this once more after `template-refresh` is merged into `main` to confirm that all the _updated_ workflows work.
